pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -67,23 +67,42 @@ public ParticipantWrapper(ParticipantWrapperPayload participantWrapperPayload, E
     }
 
     public ParticipantWrapperResult getFilteredList() {
+        return getFilteredList(false);
+    }
+
+    public ParticipantWrapperResult getFilteredList(boolean noProxyDataNeeded) {
         logger.info("Getting list of participant information");
 
         DDPInstanceDto ddpInstanceDto = participantWrapperPayload.getDdpInstanceDto().orElseThrow();
 
         if (StringUtils.isBlank(ddpInstanceDto.getEsParticipantIndex())) {
-            throw new RuntimeException("No participant index setup in ddp_instance table for " + ddpInstanceDto.getInstanceName());
+            return new ParticipantWrapperResult(0, new ArrayList<>());
```

Comment:
just show empty dashboard or pt list and no more error alert
for coloretal or voices